mkosi/distributions/debian.py

```diff
@@ -124,7 +124,6 @@ def install_packages(cls, state: MkosiState, packages: Sequence[str], apivfs: bo
         setup_apt(state, cls.repositories(state))
         invoke_apt(state, "update", apivfs=False)
         invoke_apt(state, "install", packages, apivfs=apivfs)
-        install_apt_sources(state, cls.repositories(state, local=False))
```

Comment:
The thing is, if we drop this, the user has to write the file himself, which is kind of a bad user experience unlike the other distros where you can just install a package, so I'd keep this intact for now

Reply:
Dropped the commit again.